pynucastro/networks/tests/test_compare_cxx_and_python.py

```diff
@@ -0,0 +1,88 @@
+# this creates the same network as SimpleCxxNetwork and PythonNetwork
+# and have them both compute dY/dt and compares to make sure that they
+# agree.  Note: screening is not considered.
+
+import os
+import re
+import shlex
+import subprocess
+from pathlib import Path
+
+import numpy as np
+import pytest
+from pytest import approx
+
+import pynucastro as pyna
+
+
+def run(string):
```

Comment:
How does this differ from `subprocess.run(string, capture_output=True, text=True, shell=True)`?

Reply:
I think the answer is that I copied it from a code I wrote from before subprocess.run() existed.

But it looks like using run is fine here.  I'll update it tonight